vtex-apps/currency-selector
# feat/fix: include mutation to avoid using it from binding-selector app

#### What problem is this solving?
Include `updateSalesChannel` mutation to not rely on the mutation in binding-selector and be able to remove it from peerDependencies.



#### How to test it?
![Jan-28-2022 12-42-37](https://user-images.githubusercontent.com/96049132/151541314-a4645e4e-eadd-4ae7-9bb0-a9502633ba1f.gif)

1. Go to your store and add a product to your cart.
2. Change the currency (you will see a currency symbol on the header)
3. The cart should be updated with the current currency



[Workspace](https://currencyselector--powerplanet.myvtex.com/?__bindingAddress=www.we-demostore.com/es)

#### Screenshots or example usage:



#### Describe alternatives you've considered, if any.



#### Related to / Depends on



#### How does this PR make you feel? [:link:](http://giphy.com/)



![](put .gif link here - can be found under "advanced" on giphy)
